packages/core/src/app/TimeEstimator.ts

```diff
@@ -68,7 +68,9 @@ export class TimeEstimator {
   }
 
   public reportProgress() {
-    (window as any).logProgress(this.completion.current);
+    if ((window as any).logProgress) {
```

Comment:
We could change this to `(window as any)?.logProgress?.(this.completion.current);` to make it more concise.